ibm/resource_ibm_push_notification_chrome_test.go

```diff
@@ -0,0 +1,64 @@
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMResourcePNApplicationChrome_Basic(t *testing.T) {
+	name := fmt.Sprintf("terraform_PN_%d", acctest.RandIntRange(10, 100))
+	apiKey := fmt.Sprint(acctest.RandString(45))                  // dummy value
+	websiteURL := "http://webpushnotificatons.mybluemix.net"      // dummy url
+	newWebsiteURL := "http://chromepushnotificaton.mybluemix.net" // dummy url
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
```

Comment:
Add funcitons for Testdestroy and Testexists funciton simialr to other resources...